tests/unit/helpers.js

```diff
@@ -1470,3 +1474,68 @@ export const createExperimentData = ({
 }) => {
   return { [id]: variantId };
 };
+
+const queryByClassName = (container, className) => {
+  return container.querySelectorAll(`.${className}`);
+};
+
+const getByClassName = (container, className) => {
+  const elements = queryByClassName(container, className);
+  if (!elements.length) {
+    return null;
+  }
+  if (elements.length > 1) {
+    throw new Error(`More than one element found for className ${className}`);
+  }
+  return elements[0];
+};
+
+const queryByTagName = (container, tagName) => {
+  return container.getElementsByTagName(tagName);
+};
+
+const getByTagName = (container, tagName) => {
+  const elements = queryByTagName(container, tagName);
+  if (!elements.length) {
+    return null;
+  }
+  if (elements.length > 1) {
+    throw new Error(`More than one element found for tag ${tagName}`);
+  }
+  return elements[0];
+};
```

Comment:
Do we need this helper when there is `queryByRole` already?

Reply:
I was using this to locate elements based on tags, such as `<circle>` and `<path>`, which were in my proof-of-concept, but don't appear in this PR. Those are not valid roles, so tag and role are not synonymous.

Reply:
😞  I was hoping `queryByRole` would fall back to tag names, but it wasn't clear from the docs